### PR TITLE
treewide: rename openmpt123 to libopenmpt to match the name in other distros

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/aliases.nix [hunk 1]
```diff
@@ -570,6 +570,7 @@ mapAliases ({
   openisns = open-isns; # added 2020-01-28
   openjpeg_1 = throw "openjpeg_1 has been removed, use openjpeg_2 instead"; # added 2021-01-24
   openjpeg_2 = openjpeg; # added 2021-01-25
+  openmtp123 = libopenmpt; # added 2021-09-05
```
Comment (bobby285271) [inline pkgs/top-level/aliases.nix:573]: Sorry I missed this.

```suggestion
  openmpt123 = libopenmpt; # added 2021-09-05
```

**Review by r-burns — APPROVED** [review]
Repology agrees: https://repology.org/projects/?search=openmpt

**Comment by OPNA2608** [comment]
Too large for me to nixpkgs-review rn but the change looks okay. Should the rename get mentioned in the release notes under Backward Incompatibilities?

**Comment by SuperSandro2000** [comment]
> Too large for me to nixpkgs-review rn but the change looks okay. Should the rename get mentioned in the release notes under Backward Incompatibilities?

not required. We have added an alias and except if you turned them off you will notice nothing.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 136735` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>11 packages marked as broken and skipped:</summary>
  <ul>
    <li>authenticator</li>
    <li>gnome.gnome-documents</li>
    <li>libsForQt512.dragon</li>
    <li>libsForQt512.kamoso</li>
    <li>libsForQt512.kdenlive</li>
    <li>libsForQt512.phonon-backend-gstreamer</li>
    <li>libsForQt514.dragon</li>
    <li>libsForQt514.kamoso</li>
    <li>libsForQt514.kdenlive</li>
    <li>libsForQt514.phonon-backend-gstreamer</li>
    <li>xmonad_log_applet</li>
  </ul>
</details>
<details>
  <summary>15 packages failed to build:</summary>
  <ul>
    <li>airwave</li>
    <li>aws-workspaces</li>
    <li>citrix_workspace</li>
    <li>citrix_workspace_20_04_0</li>
    <li>citrix_workspace_20_06_0</li>
    <li>citrix_workspace_20_09_0</li>
    <li>citrix_workspace_20_10_0</li>
    <li>citrix_workspace_20_12_0</li>
    <li>citrix_workspace_21_01_0</li>
    <li>citrix_workspace_21_03_0</li>
    <li>gnome.gnome-todo</li>
    <li>hunter</li>
    <li>ib-controller</li>
    <li>subtitleeditor</li>
    <li>xiphos</li>
  </ul>
</details>
<details>
  <summary>337 packages built:</summary>
  <ul>
    <li>adapta-gtk-theme</li>
    <li>alfis</li>
    <li>almanah</li>
    <li>apostrophe</li>
    <li>aravis</li>
    <li>areca</li>
    <li>astroid</li>
    <li>audacious</li>
    <li>audio-recorder</li>
    <li>balsa</li>
    <li>baresip</li>
    <li>birdfont</li>
    <li>bookworm</li>
    <li>brasero</li>
    <li>brasero-original</li>
    <li>bubblemail</li>
    <li>byzanz</li>
    <li>calls</li>
    <li>cawbird</li>
    <li>chatty</li>
    <li>chrome-gnome-shell</li>
    <li>cinnamon.cinnamon-common</li>
    <li>cinnamon.cinnamon-control-center</li>
    <li>cinnamon.cinnamon-gsettings-overrides</li>
    <li>cinnamon.cinnamon-screensaver</li>
    <li>claws-mail</li>
    <li>cog</li>
    <li>cozy</li>
    <li>denemo</li>
    <li>dropbox-cli</li>
    <li>easyeffects</li>
    <li>eclipse-mat</li>
    <li>eclipses.eclipse-committers</li>
    <li>eclipses.eclipse-cpp</li>
    <li>eclipses.eclipse-java</li>
    <li>eclipses.eclipse-jee</li>
    <li>eclipses.eclipse-modeling</li>
    <li>eclipses.eclipse-platform</li>
    <li>eclipses.eclipse-rcp</li>
    <li>eclipses.eclipse-scala-sdk</li>
    <li>eclipses.eclipse-sdk</li>
    <li>elementary-planner</li>
    <li>empathy</li>
    <li>enlightenment.rage</li>
    <li>eolie</li>
    <li>ephemeral</li>
    <li>epiphany</li>
    <li>evince</li>
    <li>evolution</li>
    <li>evolution-data-server</li>
    <li>evolution-ews</li>
    <li>evolutionWithPlugins</li>
    <li>farstream</li>
    <li>fcitx5-gtk</li>
    <li>fcitx5-with-addons</li>
    <li>feedreader</li>
    <li>foliate</li>
    <li>folks</li>
    <li>font-manager</li>
    <li>formiko</li>
    <li>fractal</li>
    <li>gajim</li>
    <li>gcompris</li>
    <li>gfbgraph</li>
    <li>giara</li>
    <li>gjay</li>
    <li>glade</li>
    <li>glom</li>
    <li>gmrender-resurrect</li>
    <li>gnome-builder</li>
    <li>gnome-feeds</li>
    <li>gnome-inform7</li>
    <li>gnome-keysign</li>
    <li>gnome-network-displays</li>
    <li>gnome-online-accounts</li>
    <li>gnome-photos</li>
    <li>gnome-podcasts</li>
    <li>gnome-recipes</li>
    <li>gnome.cheese</li>
    <li>gnome.devhelp</li>
    <li>gnome.file-roller</li>
    <li>gnome.geary</li>
    <li>gnome.gnome-applets</li>
    <li>gnome.gnome-books</li>
    <li>gnome.gnome-boxes</li>
    <li>gnome.gnome-calendar</li>
    <li>gnome.gnome-chess</li>
    <li>gnome.gnome-contacts</li>
    <li>gnome.gnome-control-center</li>
    <li>gnome.gnome-flashback</li>
    <li>gnome.gnome-initial-setup</li>
    <li>gnome.gnome-maps</li>
    <li>gnome.gnome-music</li>
    <li>gnome.gnome-notes</li>
    <li>gnome.gnome-online-miners</li>
    <li>gnome.gnome-panel</li>
    <li>gnome.gnome-session</li>
    <li>gnome.gnome-shell</li>
    <li>gnome.gnome-software</li>
    <li>gnome.gnome-sound-recorder</li>
    <li>gnome.gnome-terminal</li>
    <li>gnome.gnome-tweaks</li>
    <li>gnome.gnome-user-share</li>
    <li>gnome.gvfs</li>
    <li>gnome.nautilus</li>
    <li>gnome.nautilus-python</li>
    <li>gnome.polari</li>
    <li>gnome.pomodoro</li>
    <li>gnome.rygel</li>
    <li>gnome.sushi</li>
    <li>gnome.totem</li>
    <li>gnome.yelp</li>
    <li>gnomeExtensions.easyScreenCast</li>
    <li>gnomeExtensions.gsconnect</li>
    <li>gnomeExtensions.night-theme-switcher</li>
    <li>gnucash</li>
    <li>gnunet-gtk</li>
    <li>gnvim</li>
    <li>gnvim-unwrapped</li>
    <li>grilo-plugins</li>
    <li>gscrabble</li>
    <li>gst_all_1.gst-editing-services</li>
    <li>gst_all_1.gst-plugins-bad</li>
    <li>gst_all_1.gst-rtsp-server</li>
    <li>gst_all_1.gst-vaapi</li>
    <li>gthumb</li>
    <li>gtk4</li>
    <li>gtkd</li>
    <li>gtkmm4</li>
    <li>gtksourceview5</li>
    <li>helvum</li>
    <li>hqplayerd</li>
    <li>ipscan</li>
    <li>kaidan</li>
    <li>kgx</li>
    <li>kicad</li>
    <li>kicad-small</li>
    <li>kicad-unstable</li>
    <li>kicad-unstable-small</li>
    <li>kooha</li>
    <li>kotatogram-desktop</li>
    <li>libadwaita</li>
    <li>libgdata</li>
    <li>libgepub</li>
    <li>libopenmpt</li>
    <li>libreoffice (libreoffice-still)</li>
    <li>libreoffice-fresh</li>
    <li>libreoffice-fresh-unwrapped</li>
    <li>libreoffice-qt</li>
    <li>libreoffice-still-unwrapped</li>
    <li>libsForQt5.dragon</li>
    <li>libsForQt5.kamoso</li>
    <li>libsForQt5.kdenlive</li>
    <li>libsForQt5.phonon-backend-gstreamer</li>
    <li>libsForQt5.telepathy</li>
    <li>libsForQt512.telepathy</li>
    <li>libsForQt514.telepathy</li>
    <li>libzapojit</li>
    <li>liferea</li>
    <li>lispPackages.cl-webkit2</li>
    <li>lispPackages.nyxt</li>
    <li>lollypop</li>
    <li>luakit</li>
    <li>lutris</li>
    <li>lutris-free</li>
    <li>lutris-unwrapped</li>
    <li>mailnag</li>
    <li>mailnagWithPlugins</li>
    <li>mapmap</li>
    <li>marker</li>
    <li>mate.atril</li>
    <li>mate.mate-user-guide</li>
    <li>mavproxy</li>
    <li>megapixels</li>
    <li>meteo</li>
    <li>midori</li>
    <li>midori-unwrapped</li>
    <li>mmex</li>
    <li>mopidy</li>
    <li>mopidy-iris</li>
    <li>mopidy-local</li>
    <li>mopidy-moped</li>
    <li>mopidy-mopify</li>
    <li>mopidy-mpd</li>
    <li>mopidy-mpris</li>
    <li>mopidy-musicbox-webclient</li>
    <li>mopidy-podcast</li>
    <li>mopidy-scrobbler</li>
    <li>mopidy-somafm</li>
    <li>mopidy-soundcloud</li>
    <li>mopidy-spotify</li>
    <li>mopidy-spotify-tunigo</li>
    <li>mopidy-subidy</li>
    <li>mopidy-tunein</li>
    <li>mopidy-youtube</li>
    <li>mousai</li>
    <li>nasc</li>
    <li>newsflash</li>
    <li>nheko</li>
    <li>notejot</li>
    <li>notes-up</li>
    <li>nyxt</li>
    <li>onboard</li>
    <li>openlp</li>
    <li>openlpFull</li>
    <li>osmo</li>
    <li>pantheon.elementary-calendar</li>
    <li>pantheon.elementary-camera</li>
    <li>pantheon.elementary-capnet-assist</li>
    <li>pantheon.elementary-code</li>
    <li>pantheon.elementary-greeter</li>
    <li>pantheon.elementary-gsettings-schemas</li>
    <li>pantheon.elementary-mail</li>
    <li>pantheon.elementary-music</li>
    <li>pantheon.elementary-photos</li>
    <li>pantheon.elementary-session-settings</li>
    <li>pantheon.elementary-videos</li>
    <li>pantheon.extra-elementary-contracts</li>
    <li>pantheon.notes-up</li>
    <li>pantheon.switchboard-plug-a11y</li>
    <li>pantheon.switchboard-plug-onlineaccounts</li>
    <li>pantheon.switchboard-with-plugs</li>
    <li>pantheon.wingpanel-applications-menu</li>
    <li>pantheon.wingpanel-indicator-datetime</li>
    <li>pantheon.wingpanel-with-indicators</li>
    <li>parlatype</li>
    <li>pdfpc</li>
    <li>phosh</li>
    <li>picard</li>
    <li>pidgin</li>
    <li>pidgin-carbons</li>
    <li>pidgin-indicator</li>
    <li>pidgin-latex</li>
    <li>pidgin-mra</li>
    <li>pidgin-msn-pecan</li>
    <li>pidgin-opensteamworks</li>
    <li>pidgin-osd</li>
    <li>pidgin-otr</li>
    <li>pidgin-sipe</li>
    <li>pidgin-skypeweb</li>
    <li>pidgin-window-merge</li>
    <li>pidgin-xmpp-receipts</li>
    <li>pipelight</li>
    <li>pithos</li>
    <li>pitivi</li>
    <li>playonlinux</li>
    <li>poedit</li>
    <li>portfolio</li>
    <li>pragha</li>
    <li>printrun</li>
    <li>pulseeffects-legacy</li>
    <li>purple-discord</li>
    <li>purple-facebook</li>
    <li>purple-hangouts</li>
    <li>purple-lurch</li>
    <li>purple-matrix</li>
    <li>purple-mm-sms</li>
    <li>purple-plugin-pack</li>
    <li>purple-slack</li>
    <li>purple-vk-plugin</li>
    <li>purple-xmpp-http-upload</li>
    <li>python38Packages.humblewx</li>
    <li>python38Packages.kicad</li>
    <li>python38Packages.kivy</li>
    <li>python38Packages.wxPython_4_0</li>
    <li>python38Packages.wxPython_4_1</li>
    <li>python39Packages.humblewx</li>
    <li>python39Packages.kicad</li>
    <li>python39Packages.kivy</li>
    <li>python39Packages.wxPython_4_0</li>
    <li>python39Packages.wxPython_4_1</li>
    <li>pytrainer</li>
    <li>qgroundcontrol</li>
    <li>qt-video-wlr</li>
    <li>quisk</li>
    <li>quodlibet</li>
    <li>quodlibet-full</li>
    <li>quodlibet-without-gst-plugins</li>
    <li>quodlibet-xine</li>
    <li>quodlibet-xine-full</li>
    <li>qutebrowser</li>
    <li>qxmpp</li>
    <li>radiotray-ng</li>
    <li>recapp</li>
    <li>rednotebook</li>
    <li>remmina</li>
    <li>rhythmbox</li>
    <li>rpiplay</li>
    <li>run-scaled</li>
    <li>rymcast</li>
    <li>sayonara</li>
    <li>setzer</li>
    <li>shortwave</li>
    <li>shotwell</li>
    <li>skytemple</li>
    <li>sound-juicer</li>
    <li>sparkleshare</li>
    <li>surf</li>
    <li>surf-display</li>
    <li>swt</li>
    <li>swt_jdk8</li>
    <li>tangram</li>
    <li>tdesktop</li>
    <li>telegram-purple</li>
    <li>telepathy-farstream</li>
    <li>telepathy-haze</li>
    <li>tilix</li>
    <li>timeline</li>
    <li>tonelib-zoom</li>
    <li>toxprpl</li>
    <li>tracker-miners</li>
    <li>transcribe</li>
    <li>tunefish</li>
    <li>tuxguitar</li>
    <li>ulauncher</li>
    <li>unoconv</li>
    <li>vala-language-server</li>
    <li>vimb</li>
    <li>vimb-unwrapped</li>
    <li>vimix-gtk-themes</li>
    <li>vocal</li>
    <li>vokoscreen-ng</li>
    <li>vuze</li>
    <li>webkit2-sharp</li>
    <li>webkitgtk</li>
    <li>whatsapp-for-linux</li>
    <li>whitesur-gtk-theme</li>
    <li>wine</li>
    <li>wine-staging</li>
    <li>wineWowPackages.full</li>
    <li>winetricks</li>
    <li>wrapGAppsHook4</li>
    <li>xfce.parole</li>
    <li>xplayer</li>
    <li>xpra</li>
    <li>xpraWithNvenc</li>
    <li>yaxg</li>
  </ul>
</details>

**Pull request merged**